Samsung/r8s/res/xml/audio_assets.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<audio_assets version="1.0">
```

Comment:
What does this have to do with hardware? If you want  TouchWiz, just use stock ROM